spec/feature/user_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'UserShows', type: :system do
+  subject { User.new(name: 'John', posts_counter: 30, photo: 'https://randomuser.me/api/portraits/men/70.jpg', bio: 'Teacher from Poland.') }
+
+  before { subject.save }
+
+  describe 'index page' do
+    it 'I can see the username of all other users.' do
+      visit root_path(subject)
+      page.has_content?(subject.name)
+    end
+
+    it 'I can see the profile picture for each user.' do
+      visit root_path(subject)
+      page.has_css?('.img-fluid')
+    end
```

Comment:
- [ ] Dear @SaeqaSultani, for the user's profile picture, we need to test the presence of the image instead of the class. Please test the source image and assert the image is available.